week2/materials/homework/assignments.md

```diff
@@ -4,51 +4,73 @@
 
 Sync your changes to s3 and write down your s3 bucket url.
 
-bucket url: `?`
+bucket url: `s3://priyankas-good-greens/`
 
 **Assignment 5:**
 
 Right now, the website does not support https. Explain which other AWS service needs to be integrated in order to achieve this.
 
-service name: `?`
+service name: `CloudFront `
```

Comment:
It would be nice if some explanation would be added.

Reply:
Done 👍🏻 🙂